srcpkgs/dosbox-staging/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'dosbox-staging'
+pkgname=dosbox-staging
+version=0.77.0
+revision=1
+build_style=meson
+hostmakedepends="pkg-config ccache"
+makedepends="alsa-lib-devel libpng-devel SDL2-devel SDL2_net-devel opusfile-devel fluidsynth-devel libmt32emu-devel libatomic-devel"
```

Comment:
Please create a new line at ~80 columns.

Is `libatomic` always necessary or only for `XBPS_TARGET_NO_ATOMIC8` archs (like armv6l)?

Reply:
Done and check for `XBPS_TARGET_NO_ATOMIC8` added.